README.md

```diff
@@ -1,29 +1,33 @@
 # OCM VScode Extension
 
- "ocm-vscode-extension" helps creating OCM cluster applications. 
+The `ocm-vscode-extension` helps you quickly create OCM managed applications.
```

Comment:
"you" is unneccery

Reply:
you are right, this is resolved in https://github.com/ilan-pinto/ocm-vscode-extension/pull/27/commits/1d31e2f0c8b27325518cbc050ea11ed26b57f18c